apps/pie-docs/src/components/notification/code/props.json

```diff
@@ -127,7 +127,7 @@
                 "type": "code",
                 "item": ["{ text: string, ariaLabel?: string }"]
             },
-            "An object representing the supporting action of the notification.",
+            "An object representing the supporting action of the notification, Will only render if `leadingAction` is passed",
```

Comment:
An object representing the supporting action of the notification, **will** only render if `leadingAction` is passed"

Reply:
updated 💯